website/content/docs/internals/telemetry.mdx

```diff
@@ -377,54 +377,72 @@ These metrics relate to the supported [storage backends][storage-backends].
+| `vault.raft.transition.leader_lease_timeout`                           | Number of times quorum of nodes were not able to be contacted.                                                                                                                                                    | contact failures                  | counter |
+| `vault.raft.verify_leader`                                             | Number of times node checks whether it is still the leader or not.                                                                                                                                                | checks / interval                 | counter |
+| `vault.raft-storage.delete`                                            | Time to insert log entry to delete path.                                                                                                                                                                          | ms                                | timer   |
+| `vault.raft-storage.get`                                               | Time to retrieve value for path from FSM.                                                                                                                                                                         | ms                                | timer   |
+| `vault.raft-storage.put`                                               | Time to insert log entry to persist path.                                                                                                                                                                         | ms                                | timer   |
+| `vault.raft-storage.list`                                              | Time to list all entries under the prefix from the FSM.                                                                                                                                                           | ms                                | timer   |
+| `vault.raft-storage.transaction`                                       | Time to insert operations into a single log.                                                                                                                                                                      | ms                                | timer   |
+| `vault.raft-storage.entry_size`                                        | The total size of a Raft entry during log application in bytes.                                                                                                                                                   | bytes                             | sample  |
+| `vault.raft_storage.bolt.freelist.free_pages`                          | Number of free pages in the freelist.                                                                                                                                                   | bytes                             | sample  |
```

Comment:
Are these really all samples?  Most of them are actually gauges I think.

Reply:
Whoops, copy/paste error on my part 🤦 . I was so focused on the formatting, I missed that. I'll get it fixed.

Reply:
f6c4a68f4f83657b5660647c705f9fc5f9b2da9f